docs/DeveloperGuide.md

```diff
@@ -514,24 +514,92 @@ The following sequence diagram shows how the event show operation works:
   * Pros: Might be easier to implement as the `EventListPanel` will already have access to the event information as it has `filteredEventList` as one of its fields.
   * Cons: Depletes the user experience as they will have to use the `elist` command, followed by the `eshow` command if they want to see the event information of another event.
 
-### \[In progress\] Delete the event from a list of event
+### Delete the event from a list of events
```

Comment:
`Delete the event from a list of events` -> `Delete an event from a list of events`